inc/spbc-settings.php

```diff
@@ -99,6 +99,11 @@ function spbc_settings__register()
         }
     }
 
+    $company_name = 'ClaenTalk';
```

Comment:
First one - there is an typo. Second one - please, get company name by default from [`State`](https://github.com/CleanTalk/security-malware-firewall/blob/master/lib/CleantalkSP/SpbctWP/State.php#L211)